aws/logs_monitoring/README.md

```diff
@@ -100,17 +100,14 @@ If you can't install the Forwarder using the provided CloudFormation template, y
 
 ## AWS PrivateLink Support
 
-You can run the Forwarder in a VPC by using AWS PrivateLink.
+You can run the Forwarder in a VPC by using AWS PrivateLink to connect to Datadog. Note that AWS PrivateLink can only be configured with Datadog organizations in the Datadog US region.
```

Comment:
I think in our docs we call that the "US site". Might be good to clarify ("that is, datadoghq.com, not .eu")